pkg/bpf/prog.go

```diff
@@ -85,21 +88,25 @@ func (t ProgType) String() string {
 		return "Raw tracepoint"
 	case ProgTypeCgroupSockAddr:
 		return "Cgroup sockaddr"
+	case ProgTypeLwtSeg6Local:
+		return "LWT seg6local"
+	case ProgTypeLircMode2:
+		return "LIRC"
+	case ProgTypeSkReusePort:
+		return "Socket SO_REUSEPORT"
 	}
 
 	return "Unknown"
 }
 
 // attrProg holds values from the upstream struct union for BPF_*_GET_*_ID.
```

Comment:
The URL link to the Linux code was kept for two reasons:
1. Link to the relevant code so that anyone can find it and compare the latest definitions against these definitions.
1. Link to the specific version that this code has been synced with.

Would you mind updating the link instead of removing it?

Same with the comments below.

Reply:
Done, PTAL.